infra_event_notifier/backends/datadog.py

```diff
@@ -0,0 +1,44 @@
+import time
+import json
+
+import urllib.request
+
+# from urllib.request import Request, urlopen
+from typing import Mapping
+
+
+def notify_datadog(
```

Comment:
Something about naming:
As this is a specific type of datadog notification (datadog event) I would call it `send_event`. No need to call the method `notify_datadog` this method is in the datadog backend, so it will necessarily be a datadog call.